CONTRIBUTING.rst

```diff
@@ -84,6 +84,16 @@ The main one is the SunPy repository with where all the known `issues`_ with Sun
 Each issue should have a series of labels that provide information about the nature of the issue.
 If you find an issue you'd like to work on, please make sure to add a comment to let people know that you are working on it! This will make it less likely that effort is duplicated.
 
+.. note::
+
+    sunpy is Free and open-source software (FOSS), under the BSD-2 license. By contributing you are stating that you have the right to and agree to, have your work distributed under the terms of this license.
+
+    If you work at a university or institution, you will need to check if you can contribute code you have written at work.
```

Comment:
This should probably be slightly expanded to include everyone on work time no matter who ever their employer is, although obviously it makes sense for us to focus on unis.

Reply:
Also we might want to consider adding a link to this or something in the PR template?

Reply:
Added, see what you think.